spatial-tracking-explainer.md

```diff
@@ -257,6 +278,16 @@ It is expected that developers will often choose to preview `immersive` experien
 #### Unbounded to Bounded 
 When building an experience that is predominantly based on an `unbounded` reference space, developers may occasionally choose to switch to a `bounded` reference space. For example, a whole-home renovation experience might choose to switch to a bounded reference space for reviewing a furniture selection library.  If necessary to continue displaying content belonging to the previous reference space, developers may call the `XRFrame`'s `getPose()` method to re-parent nearby virtual content to the new reference space.
 
+#### Viewer space
```

Comment:
This whole section is incorrect now :(

Reply:
The concept of passing the `viewer` reference space to `getPose` is now covered by the "Viewer reference space" section, so I just removed this section.